rules/solid/tests/Rector/If_/ChangeAndIfToEarlyReturnRector/Fixture/skip_if_in_foreach.php.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Rector\SOLID\Tests\Rector\If_\ChangeAndIfToEarlyReturnRector\Fixture;
+
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Name;
+
+class SkipIfInForeachClass
+{
+    private function refactorStmts(array $nodes): void
+    {
+        foreach ($nodes as $node) {
+            if ($node instanceof Name && $node instanceof Identifier) {
+                $this->processNameOrIdentifier($node);
+            }
+
+            return;
+        }
+
```

Comment:
@TomasVotruba This situation looks crazy but also can be handled in the next PR.